dm/ui/src/layouts/SiderMenu.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import { useNavigate, useLocation } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+
+import { Menu } from '~/uikit'
+import {
+  DashboardOutlined,
+  ExportOutlined,
+  ClusterOutlined,
+} from '~/uikit/icons'
+
+const { SubMenu } = Menu
+
+const SiderMenu: React.FC<{
+  collapsed: boolean
+}> = ({ collapsed }) => {
+  const [t] = useTranslation()
+  const navigate = useNavigate()
+  const loc = useLocation()
+
+  const items = [
+    { itemKey: '/', text: 'Dashboard', icon: <DashboardOutlined /> },
+    {
+      itemKey: '/migration',
+      text: t('migration'),
+      icon: <ExportOutlined />,
+      items: [
+        {
+          text: t('task list'),
+          itemKey: '/migration/task',
+        },
+        {
+          text: t('source list'),
+          itemKey: '/migration/source',
+        },
+        {
+          text: t('task config'),
+          itemKey: '/migration/task-config',
+        },
+        {
+          text: t('sync detail'),
+          itemKey: '/migration/sync-detail',
+        },
+      ],
+    },
+    {
+      itemKey: '/cluster',
+      text: t('cluster management'),
+      icon: <ClusterOutlined />,
+      items: [
+        {
+          text: t('member list'),
+          itemKey: '/cluster/member',
+        },
+        {
+          text: t('relay log'),
+          itemKey: '/cluster/relay-log',
+        },
+      ],
+    },
+  ]
+  return (
+    <div>
+      <div className="flex p-4 justify-center">
+        <img
+          src="https://internals.tidb.io/uploads/default/original/1X/4fde143c268ccde514d0c93e420b8a2304fc2033.png"
```

Comment:
This image can't be accessed inside a private network, it is better to put in local.

Reply:
sure